recipes/sccache/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "0.2.12" %}
+
+package:
+  name: sccache
+  version: {{ version }}
+
+source:
+  url: https://github.com/mozilla/sccache/archive/{{ version }}.tar.gz
+  sha256: 591a82ddbc2e970630a9426c78c25cbc52c3261b06d57cb4e1f11ab8008629fa
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - rust
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - openssl
```

Comment:
``openssl`` should not be here, could you please delete it?

Reply:
I think ``rust`` should be in ``host`` and ``run`` requirements, am I right?

Reply:
You don't need `rust` to run, it generates a static binary (but linking to openssl dinamically). Is `build` requirements only for `{{ compiler }}` deps?